contractcourt/chain_arbitrator.go

```diff
@@ -142,7 +142,7 @@ type ChainArbitratorConfig struct {
 
 	// DisableChannel disables a channel, resulting in it not being able to
 	// forward payments.
-	DisableChannel func(wire.OutPoint) error
+	DisableChannel func(wire.OutPoint, bool) error
```

Comment:
Good call! In retrospect, `ChainArbitrator` should be calling this field with `manual = true`, so I switched it back to the old type:

```
DisableChannel: func(chanPoint wire.OutPoint) error {
	return s.chanStatusMgr.RequestDisable(chanPoint, false)
},
```

As for the enum, I left it alone because I think "human" and "not human" are the only two cases we have to deal with for now (though watch as I eat my words sometime soon...)